CONTRIBUTING.rst

```diff
@@ -0,0 +1,57 @@
+Contributing
+############
+
+Contributions are welcome and much appreciated!
+
+Development
+-----------
+
+To setup `ConferenceScheduler` for local development:
+
+1. Fork the repository on Github `<https://github.com/https://github.com/PyconUK/ConferenceScheduler/fork>`_
+2. Clone your fork to your local machine::
+
+    git clone git@github.com:your_name_here/ConferenceScheduler.git
+
+3. Create a branch for your work::
+
+    git checkout -b name-of-your-branch
+
+4. Commit your changes and push your branch to Github::
+
+    git add -A
```

Comment:
Perhaps we don't need to go in to the details here? (I personally do not recommend `git add -A` and `git commit -m ...`.